.github/workflows/transifex_update_english.yml

```diff
@@ -14,15 +14,17 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0
+          ref: transifex-synchronization-source
 
       - name: Setup Git User
         run: |
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config user.name "github-actions[bot]"
 
-      - name: Transifex Rebase
+      - name: Update English Source to Transifex
         run: |
-          git checkout transifex-synchronization-source
-          git reset origin/master --hard
-          git push origin transifex-synchronization-source --force-with-lease
+          git fetch origin master
```

Comment:
Why do this dance instead just overriding whole `transifex-synchronization-source` with what is on master?

Reply:
My attempt to make the workflow cleaner, once a week English goes master -> transifex, and translations go transifex -> master on demand.

I guess the only edge case is if a translation PR is merged while this action is running, but since that happens in the middle of the night and those PRs are merged manually, that shouldn't be a concern?

Reply:
Isn't just overriding what is on `transifex-synchronization-source` with `master` cleaner? Then you can also just compare branches in github UI very easily, you don't have those fully diverging branches.

The only race condition is if this workflow is run just after PR gets merged but before the workflow that commits to master is run, but that shouldn't be any issue as it will be just syncer back from transifex again, so: no lose of any data and *extremely* unlikely to ever happen.